### PR TITLE
Added faster and more reliable string comparisons

### PR DESCRIPTION
**Pull request by linux-fryer**
Use it thanx also changed it to c++17 bc ya

### CODE REVIEW

**Comment by checkraisefold** [comment]
What if there's an exe named RobloxPlayerBetaTool running

**Comment by linux-fryer** [comment]
Don't care didn't ask

File: src/dllmain.cpp [hunk 1]
```diff
@@ -155,7 +156,7 @@ uint8_t mumble_initPositionalData(const char* const * programNames, const uint64
 	// Check if game is open
 	for (int i = 0; i < programCount; ++i)
 	{
-		if (strcmp(programNames[i], "RobloxPlayerBeta") == 0 || strstr(programNames[i], "RobloxPlayerBeta.") != nullptr)
+		if (std::string_view(programNames[i]).find("RobloxPlayerBeta"))
```
Comment (checkraisefold) [inline src/dllmain.cpp:159]: Add a . to the end of RobloxPlayerBeta

**Review by checkraisefold — CHANGES_REQUESTED** [review]

File: src/dllmain.cpp (hunk 1 above)
Comment (linux-fryer) [reply src/dllmain.cpp:159]: kys

**Pull request closed without merging**